docs/content/python/getting-started/application/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Add Oso to Your Python App
+weight: 2
+description: |
+    In this guide, we'll cover the basics of adding Oso to a Python application
+    to enforce authorization. Start here to understand the basics of using
+    Oso. Then, jump into our more detailed how to section on topics that are
+    important to you.
+aliases:
+  - /getting-started/application/index.html
+---
+
+# Add Oso to Your Python App
+
+In this guide, we'll cover the basics of adding Oso to your Python application
+to enforce authorization. Start here to understand the basics of using
+Oso. Then, jump into our more detailed how to section on topics that are
+important to you.
+
+To use Oso, you'll:
+
+- [Model your authorization policy](model) in Polar using *resources*.
```

Comment:
Yeah, agreed. Updated & added policy into the description because I feel like mentioning it at this top level is important (I want someone to understand the policy is a key part of using Oso)